### PR TITLE
Add methods assigned to SM

### PR DESCRIPTION
**Pull request by steve-meyers**
- [X] Add highest_total_score method

- [X] Add lowest_total score method

- [X] Add biggest_blowout method

- [X] Add integer conversion to Game class attributes

### CODE REVIEW

**Comment by javier-aguilar** [comment]
Nice catch with the game class integers

File: test/stat_tracker_test.rb [hunk 1]
```diff
@@ -22,6 +22,20 @@ def test_game_collection
...
+
+  def test_it_show_lowest_total_score
+    assert_equal 3, @stat_tracker.lowest_total_score
+  end
+
+  def test_it_show_biggest_blowout
+    assert_equal 2, @stat_tracker.biggest_blowout
+  end
+
+
```
Comment (javier-aguilar) [inline test/stat_tracker_test.rb:37]: I would recommend removing the extra lines between test_it_show_biggest_blowout and test_calculates_percentage_home_wins

**Comment by mlynch5187** [comment]
Lookin' good.

File: lib/game.rb [hunk 2]
```diff
@@ -12,14 +12,14 @@ class Game
...
     @type           = game_params[:type]
     @date_time      = game_params[:date_time]
-    @away_team_id   = game_params[:away_team_id]
-    @home_team_id   = game_params[:home_team_id]
-    @away_goals     = game_params[:away_goals]
-    @home_goals     = game_params[:home_goals]
+    @away_team_id   = game_params[:away_team_id].to_i
+    @home_team_id   = game_params[:home_team_id].to_i
+    @away_goals     = game_params[:away_goals].to_i
+    @home_goals     = game_params[:home_goals].to_i
```
Comment (Shock253) [inline lib/game.rb:22]: Thanks for changing this, I missed that in my tests

**Review by Shock253 — COMMENTED** [review]
Looks good, I see no problems

**Pull request merged**